.github/workflows/triage.yml

```diff
@@ -0,0 +1,28 @@
+name: Auto triage based on the component label in issue
```

Comment:
What do you think about consolidating this job into the same workflow as https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/add-untriaged.yml

Reply:
Updated this in latest commit. Example issue - https://github.com/mch2/OpenSearch/issues/275